taskweaver/role/role.py

```diff
@@ -99,6 +121,61 @@ def reply(self, memory: Memory, **kwargs: ...) -> Post:
     def close(self) -> None:
         self.logger.info(f"{self.alias} closed successfully")
 
+    def load_experience(
+        self,
```

Comment:
these methods seems have no much direct relationship with each role, maybe could move to experience generator?

Reply:
answered above